src/main/java/com/redhat/devtools/intellij/quarkus/QuarkusModuleUtil.java

```diff
@@ -152,6 +150,34 @@ private static boolean isQuarkusModule(VirtualFile file, Project project) {
     }
 
     public static @Nullable VirtualFile getModuleDirPath(@NotNull Module module) {
-        return LocalFileSystem.getInstance().findFileByPath(ModuleUtilCore.getModuleDirPath(module));
+        VirtualFile[] roots = getContentRoots(module);
+        if (roots.length > 0) {
+            return roots[0];
+        }
+        return VfsUtil.findFileByIoFile(new File(ModuleUtilCore.getModuleDirPath(module)), true);
```

Comment:
return ProjectUtil.guessModuleDir(module);

Reply:
Are you sure? Need to debug that to check if we have the same result.

Reply:
But this code is just a fallback, because generally project have contents root.

Reply:
no I thought ModuleUtilCore.getModuleDirPath was internal. But it looks fine to use

Reply:
I would prefer using VfsUtil.findFileByIoFile(new File(ModuleUtilCore.getModuleDirPath(module)), true); which have the same behavior than previous version.

Are you OK with that or do you prefer I use ProjectUtil.guessDir

Reply:
yes I said it's fine to use

Reply:
> no I thought ModuleUtilCore.getModuleDirPath was internal. But it looks fine to use

It is `module.getModuleFilePath()` which is internal and ModuleUtilCore.getModuleDirPath use it.